src/components/Header.tsx

```diff
@@ -0,0 +1,234 @@
+    font-size: 15px;
+    line-height: 150%;
+    letter-spacing: -0.04px;
+    text-transform: uppercase;
+    color: #303f9f;
+  }
+`;
+
+const HeaderHomePage = () => {
+  return (
+    <Header>
+      <ContainerHeader>
+        <Brand>
+          <Logo src={logo} alt="" />
+          <Typography
+            variant="h6"
+            sx={{
+              width: '420px',
+              height: '32px',
+              fontFamily: 'Roboto',
+              fontStyle: 'normal',
+              fontWeight: '700',
+              fontSize: '19px',
+              lineHeight: '160%',
+              color: '#FFFFFF'
+            }}>
+            CỔNG THÔNG TIN TIÊM CHỦNG COVID-19
+          </Typography>
+        </Brand>
+        <Menu>
+          <ItemHome>
+            <Typography
+              variant="body1"
+              sx={{
+                width: '71px',
+                height: '24px',
+                fontFamily: 'Roboto',
+                fontStyle: 'normal',
+                fontWeight: '500',
+                fontSize: '16px',
+                lineHeight: '150%',
+                letterSpacing: '-0.04px',
+                color: '#FFFFFF'
+              }}>
+              Trang chủ
+            </Typography>
+          </ItemHome>
+          <ItemInject>
+            <Typography
+              variant="body1"
+              sx={{
+                width: '93px',
+                height: '24px',
+                fontFamily: 'Roboto',
+                fontStyle: 'normal',
+                fontWeight: '500',
+                fontSize: '16px',
+                lineHeight: '150%',
+                letterSpacing: '-0.04px',
+                color: '#FFFFFF'
+              }}>
+              Đăng ký tiêm
+            </Typography>
+          </ItemInject>
+          <ItemResearch>
+            <Typography
+              variant="body1"
+              sx={{
+                width: '71px',
+                height: '20px',
+                fontFamily: 'Roboto',
+                fontStyle: 'normal',
+                fontWeight: '500',
+                fontSize: '12px',
+                letterSpacing: '-0.04px',
+                color: '#FFFFFF'
+              }}>
+              Tra cứu
+            </Typography>
+            <KeyboardArrowDownIcon htmlColor="white" />
+          </ItemResearch>
+          <ItemDocs>
+            <Typography
+              variant="body1"
+              sx={{
+                width: '71px',
+                height: '24px',
+                fontFamily: 'Roboto',
+                fontStyle: 'normal',
+                fontWeight: '500',
+                fontSize: '16px',
+                lineHeight: '150%',
+                letterSpacing: '-0.04px',
+                color: '#FFFFFF'
+              }}>
+              Tài liệu
+            </Typography>
+          </ItemDocs>
+          <ItemLogin>
+            <ButtonLogin>
+              <Link to="/login">Đăng nhập</Link>
+            </ButtonLogin>
```

Comment:
Should not place the link inside Button, it will not go full the container button space
Please using component props to customize Button Link. Read more API Ref: https://mui.com/material-ui/api/button/